### PR TITLE
#1674 Creating the IEEE30 test cases (adaptation of Figueras' work)

### LINKED ISSUE #1674: IEEE 30 Bus System test cases
Hi, I attach the IEEE 30 bus system test cases: i) Three-phase fault ii) Line disconnection ii) Generator disconnection. I add a brief report that summarises the results and models used. Also, in the iidm file, the p1, q1, q2, p2 data is missing for transformers and lines. 

[IEEE30_Bus_System.pdf](https://github.com/dynawo/dynawo/files/6517115/IEEE30_Bus_System.pdf)
[IEEE30.zip](https://github.com/dynawo/dynawo/files/6517117/IEEE30.zip)

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
closes #1674

### CODE REVIEW

File: nrt/data/IEEE30/cases.py
Comment (joyelfeghali) [inline nrt/data/IEEE30/cases.py:1]: We can put them all in a folder called BasicTestCases and rename the cases to NodeFault, DisconnectGroup and DisconnectLine (similar to IEEE14 and IEEE57 nrt)

File: nrt/data/IEEE30/IEEE30_Fault/IEEE30.crv
Comment (joyelfeghali) [inline nrt/data/IEEE30/IEEE30_Fault/IEEE30.crv:1]: We can also add description folder and description file in latex (see the description pdf in the issue #1674 )

**Review by joyelfeghali — CHANGES_REQUESTED** [review]

File: nrt/data/IEEE30/IEEE30_BasicTestCases/IEEE30_NodeFault/IEEE30.par [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+    <par type="DOUBLE" name="Pm_ValueIn" value="0"/>
+    <reference name="generator_P0Pu" origData="IIDM" origName="p_pu" type="DOUBLE"/>
+    <reference name="generator_Q0Pu" origData="IIDM" origName="q_pu" type="DOUBLE"/>
+    <reference name="generator_U0Pu" origData="IIDM" origName="v_pu" type="DOUBLE"/>
+    <reference name="generator_UPhase0" origData="IIDM" origName="angle_pu" type="DOUBLE"/>
+  </set>
+  <set id="Fault">
+    <par type="DOUBLE" name="fault_RPu" value="0.0001"/>
+    <par type="DOUBLE" name="fault_XPu" value="0.0001"/>
+    <par type="DOUBLE" name="fault_tBegin" value="21"/>
```
Comment (joyelfeghali) [inline nrt/data/IEEE30/IEEE30_BasicTestCases/IEEE30_NodeFault/IEEE30.par:354]: value="1"

File: nrt/data/IEEE30/IEEE30_BasicTestCases/IEEE30_NodeFault/IEEE30.par [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+    <par type="DOUBLE" name="fault_tEnd" value="21.1"/>
```
Comment (joyelfeghali) [inline nrt/data/IEEE30/IEEE30_BasicTestCases/IEEE30_NodeFault/IEEE30.par:355]: value="1.1"

File: nrt/data/IEEE30/IEEE30_BasicTestCases/description/IEEE30.tex [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+Generator 1 is electrically close to generator 2. Therefore, when the fault is applied at bus 2, the reactive power injection of generator 1 contributes to limit the voltage dip on bus 1. The active power injection of generator 1 increases to compensate the null injection coming from generator 2 and after the fault clears, oscillates about 8 seconds before returning to its initial value.
+
+\begin{figure}[H]
+  \centering
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - g2 - P.png}}
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - g2 - Q.png}}
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - g2 - omega.png}}
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - bus 2 - voltage.png}}
+    \caption{Generator 1 response to three-phase fault}
```
Comment (joyelfeghali) [inline nrt/data/IEEE30/IEEE30_BasicTestCases/description/IEEE30.tex:104]: Change the png file, it's g1 and bus 1. Thanks.

**Review by joyelfeghali — CHANGES_REQUESTED** [review]

File: nrt/data/IEEE30/IEEE30_BasicTestCases/description/IEEE30.tex (hunk 3 above)
Comment (ErwanGuichard49) [reply nrt/data/IEEE30/IEEE30_BasicTestCases/description/IEEE30.tex:104]: Corrected

File: nrt/data/IEEE30/IEEE30_BasicTestCases/IEEE30_NodeFault/IEEE30.par [hunk 4]
```diff
@@ -351,8 +351,8 @@
   <set id="Fault">
     <par type="DOUBLE" name="fault_RPu" value="0.0001"/>
     <par type="DOUBLE" name="fault_XPu" value="0.0001"/>
-    <par type="DOUBLE" name="fault_tBegin" value="21"/>
-    <par type="DOUBLE" name="fault_tEnd" value="21.1"/>
+    <par type="DOUBLE" name="fault_tBegin" value="1"/>
+    <par type="DOUBLE" name="fault_tEnd" value="1.1"/>
```
Comment (joyelfeghali) [inline nrt/data/IEEE30/IEEE30_BasicTestCases/IEEE30_NodeFault/IEEE30.par:355]: nrt is OK after changing these values?

Comment (ErwanGuichard49) [reply nrt/data/IEEE30/IEEE30_BasicTestCases/IEEE30_NodeFault/IEEE30.par:355]: It has just been updated.

**Review by joyelfeghali — APPROVED** [review]

**Pull request merged**